packages/docusaurus-theme-classic/src/theme/DocVersionBadge/index.tsx

```diff
@@ -22,7 +23,8 @@ export default function DocVersionBadge({
           ThemeClassNames.docs.docVersionBadge,
           'badge badge--secondary',
         )}>
-        Version: {versionMetadata.label}
+        <Translate id="theme.docs.versionBadge.label">Version: </Translate>
```

Comment:
what about using a translate param for the label?

I'm not 100% sure `Version: {label}` works for all locales so it's worth using the more flexible option